app/src/main/java/com/android/solvit/provider/ui/profile/ProfessionalProfile.kt

```diff
@@ -0,0 +1,324 @@
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.draw.shadow
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.TextUnit
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import coil.compose.AsyncImage
+import com.android.solvit.R
+import com.android.solvit.seeker.model.provider.ListProviderViewModel
+import com.android.solvit.shared.model.provider.Provider
+import com.android.solvit.shared.ui.authentication.VerticalSpacer
+import com.android.solvit.shared.ui.navigation.NavigationActions
+import com.android.solvit.shared.ui.navigation.Route
+import com.google.firebase.Firebase
+import com.google.firebase.auth.auth
+
+@Composable
+fun ProfessionalProfileScreen(
+    listProviderViewModel: ListProviderViewModel =
+        viewModel(factory = ListProviderViewModel.Factory),
+    navigationActions: NavigationActions
+) {
+  val userId = Firebase.auth.currentUser?.uid ?: "-1"
+  val provider =
+      listProviderViewModel.providersList.collectAsState().value.first { it.uid == userId }
+  Column(modifier = Modifier.fillMaxSize().background(Color.White).testTag("background")) {
+    ProfileHeader(navigationActions = navigationActions, provider = provider)
+    VerticalSpacer(10.dp)
+    JobsDoneSection()
+    VerticalSpacer(10.dp)
+    StatsSection(provider = provider)
+  }
+}
+
+@Composable
+fun ProfileHeader(navigationActions: NavigationActions, provider: Provider) {
+  Row(modifier = Modifier.fillMaxWidth()) {
+    Column(
+        modifier =
+            Modifier.background(Color(0, 121, 107, 100)).height(400.dp).padding(8.dp).weight(1f),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.Top) {
+          Column(
+              modifier = Modifier.align(Alignment.Start),
+              horizontalAlignment = Alignment.Start,
+              verticalArrangement = Arrangement.Bottom) {
+                Box {
+                  IconButton(
+                      onClick = { navigationActions.navigateTo(Route.REQUESTS_FEED) },
+                      modifier = Modifier.testTag("backButton")) {
+                        Icon(
+                            Icons.AutoMirrored.Filled.ArrowBack,
+                            contentDescription = "Back",
+                            modifier = Modifier.size(24.dp),
+                            tint = Color(239, 70, 55))
+                      }
+                }
+              }
```

Comment:
Nit: I understand that this is done to follow the Figma, but it is a little laborious, and I would argue that the gain in visual design does not outweigh the loss of code clarity. I suggest creating an explicit top bar with the back arrow, the screen name, and maybe a menu icon to access other features in the future.